hack/jenkins/update-jobs.sh

```diff
@@ -42,4 +42,4 @@ fi
 
 docker exec job-builder git checkout master
 docker exec job-builder git pull
-docker exec job-builder jenkins-jobs test ${config_dir}
+docker exec job-builder jenkins-jobs update ${config_dir}
```

Comment:
Jenkins has [this](https://wiki.jenkins-ci.org/display/JENKINS/Mail+Watcher+Plugin) for sending off an email when the job config changes. It has to be applied per-job, though.